### PR TITLE
two_fer: sync with canonical data

### PR DESCRIPTION
**Pull request by Grociu**
part of #1762 

- changed the argument of test function to None from empty string.
- rewritten the example to fit the new test

### CODE REVIEW

File: exercises/two-fer/example.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-def two_fer(name=""):
-    if not name.strip():
```
Comment (Grociu) [inline exercises/two-fer/example.py]: This no longer worked with None as the argument for test_no_name_given

File: exercises/two-fer/two_fer_test.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 class TwoFerTest(unittest.TestCase):
     def test_no_name_given(self):
-        self.assertEqual(two_fer(), 'One for you, one for me.')
+        self.assertEqual(two_fer(None), 'One for you, one for me.')
```
Comment (Grociu) [inline exercises/two-fer/two_fer_test.py]: As per canonical data switched null to None as this is the equivalent.

**Comment by cmccandless** [comment]
Merged; thanks for working on this!

**Comment by yawpitch** [comment]
I'm going to say this change should **not** have been made.

It makes no sense to force students two write an entirely un-Pythonic solution due to what is effectively an oversight in the canonical data. It is insensible to make a `two_fer(None)` call rather than a `two_fer()` call when attempting to teach default arguments, as the default argument can _never_ be involved with the tests as rewritten.

**Comment by SleeplessByte** [comment]
FWIW: `None` might be equivalent to `null` when that `null` is _JavaScript_, but I would say that normally we would have used `undefined` in the JS equivalent of this canonical data. `null` from JSON doesn't blindly translate.

**Pull request merged**